composer.json

```
@@ -37,6 +37,9 @@
     "ergebnis/phpstan-rules": "^1.0",
     "vimeo/psalm": "^4.23"
   },
+  "support": {
+    "source": "https://github.com/sgerrand/duffel-api-php.git"
```

```suggestion
    "source": "https://github.com/duffelhq/duffel-api-php.git"
```

Guessing you probably want to point to the Duffel one?

---

Very much so. Thanks again for the eagle :eye:.